src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePoPCryptoProvider.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography;
+using Microsoft.Identity.Client.AuthScheme.PoP;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.Platforms.netcore
```

Comment:
As @henrik-me  points out, you should be able to reuse this code between .net core and .net classic (and .net standard?!). We have a directory - Platforms/Shared with common implementations that you can put it.

Reply:
ah, ok. that makes more sense. Ill refactor